### ISSUE TITLE
Support for installing cirrus in conda environments

### ISSUE THREAD

**Issue by evansde77** [evansde77/cirrus#158]
Given a lot of people use conda, should look into supporting installing via conda. 

Initial tests look promising:

Tests using container:

```bash
docker pull continuumio/anaconda
docker run -ti continuumio/anaconda:latest 
```

on the container:

```bash
conda create -n venv
source activate venv
pip install cirrus-cli
git config --global cirrus.github-username evansde77
git config --global alias.cirrus  "! /opt/conda/bin/cirrus"     
```

Unfortunately the git alias doesnt seem to work:

```bash
git cirrus hello
...
fatal: While expanding alias 'cirrus': ' /opt/conda/bin/cirrus': No such file or directory
```

So need to figure out whats going on there. Seems pretty close though. 
Tried messing with the alias a bit to no avail, also verified executablity.

**Comment by evansde77** [comment]
Patching delegate to get this to work, looks like I need to tweak the other entrypoints to accept the initial args for this from sys.argv

```python
def main():
    """
    _main_

    response to the cirrus <verb> command
    Extracts the available verbs that are installed as
    entry points by setup.py as cirrus_commands

    """ 
    commands = {}
    for script in pkg_resources.iter_entry_points(group="cirrus_commands"):
        comm = str(script).split(" = ", 1)[0]
        commands[comm] = script.load()



    # switch to the current GIT_PREFIX working dir
    old_dir = os.getcwd()
    os.chdir(os.path.abspath(os.environ.get('GIT_PREFIX', '.')))

    try:
        args = sys.argv[1:]
        if len(args) == 0 or args[0] == '-h':
            # missing command or help
            print format_help(commands)
            exit_code = 0
        else:
            if args[0] not in commands:    
                msg = "Unknown command: {}".format(args[0])
                print msg
                print format_help(commands)
                exit_code = 127
            else:
                exit_code = commands[args[0]]()
    finally:
        # always return to previous dir
        os.chdir(old_dir)
    return exit_code
```

**Comment by evansde77** [comment]
released in 0.2.1
